src/convertToObject.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleDeclarations = sourceString.split(';').filter(Boolean);
```

Comment:
Can make func splitter which takes separator. Use in this line and 9 line ( not critical)